scripts/generate-and-apply-external-dns-yaml.sh

```diff
@@ -55,6 +55,17 @@ else # Using AWS Route53
   yq e -i '.externalDns.deployment.args[3] = env(DOMAIN_FILTER)' generated/$CLUSTER_NAME/external-dns/external-dns-data-values.yaml
   yq e -i '.externalDns.deployment.args[9] = env(HOSTED_ZONE_ID)' generated/$CLUSTER_NAME/external-dns/external-dns-data-values.yaml
 
+  # Perform special processing to handle Cloudgate use case where session tokens are used 
+  if [ -z "$AWS_SESSION_TOKEN" ]; then
```

Comment:
Can't assume user is aws as ias.  Must test for params.yaml .iaas = aws

Reply:
> Can't assume user is aws as ias. Must test for params.yaml .iaas = aws

@doddatpivotal 

Aren't we assuming here that the user is using AWS for DNS?  If so we still should probably check to see if the session token is present for use with route53,  the problem is if they are using route53 with a different IAAS,  the previous script will not have edited the policies correctly.